### PR TITLE
[Refactor:TAGrading] Remove old TA grading interface

### PR DESCRIPTION
**Pull request by mukul-kmr-jha**
### Please check if the PR fulfills these requirements:

* [x] The PR title and message follows our [guidelines](https://submitty.org/developer/how_to_contribute)
* [ ] Tests for the changes have been added/updated (if possible)
* [ ] Documentation has been updated/added if relevant

### What is the current behavior?
Currently, we have two TA grading interfaces the legacy (old) Interface and the new one. The new TA grading interface is in the beta stage and can be checked out by tapping out the`tag-icon` provided on the old interface toolbar. (It has `/beta` in its URL endpoint)

### What is the new behavior?
Removed support for the old TA-grading interface and set the new interface as the default one.

### CODE REVIEW

**Comment by mukul-kmr-jha** [comment]
Hi @bmcutler , really sorry for no catch up on this one, I'm going to fix the merge conflicts and push the changes ASAP, thanks!

**Comment by bmcutler** [comment]
@mukul-kmr-jha 
Are you able to help with the merge conflicts?
We would like to merge this ASAP.  Thanks!

**Comment by mukul-kmr-jha** [comment]
Sure, adding the changes

.

**Comment by bmcutler** [comment]
@shailpatels 
This PR seems to be passing the github action php lint but failing the travis php lint.
Can you investigate?

**Comment by shailpatels** [comment]
@bmcutler  hmm not sure yet, travis and gh actions seem to use the same version for phpcs and phpstan so not sure what the difference might be.

@MasterOdin any ideas ? Do you think its the version of PHP travis is running against, I think on github its grabbing whatever's the latest

**Review by emaicus — APPROVED** [review]
I applied a patch to remove the linting warnings/some errors that were being thrown on the TA grading page. I think that this PR looks good to go.

File: site/app/views/grading/ElectronicGraderView.php [hunk 1]
```diff
@@ -1062,14 +1059,13 @@ public function hwGradingPage(Gradeable $gradeable, GradedGradeable $graded_grad
...
             }
         }
-        if ($showNewInterface) {
-            $return .= <<<HTML
-                    </div>
-                </div>
-            </div>
-		</div>
+
+        $return . <<<HTML
```
Comment (MasterOdin) [inline site/app/views/grading/ElectronicGraderView.php:1063]:
```suggestion
        $return .= <<<HTML
```

**Review by MasterOdin — CHANGES_REQUESTED** [review]

**Comment by MasterOdin** [comment]
@bmcutler, @shailpatels The difference between Travis and GitHub actions is that the former is using PHP 7.2 and the latter is using PHP 7.4. In PHP 7.3, ("flexible heredocs")[https://www.php.net/manual/en/migration73.new-features.php] were introduced which lifted some of the rigorous requirements of heredocs of prior PHP versions, including that the closing tag had to have no indentation. In PHP 7.2 and before, if the closing tag was indented, it was not considered as a closing tag, and so would keep parsing till it found one. This then makes the file end up in a very weird state for phpstan / phpcs to scan and so we get a ton of weird error messages. In PHP 7.3+, you can now have indentation.

The block that @emaicus edited in his commit had this indented heredoc closing tag:
https://github.com/Submitty/Submitty/blob/fdd96da52b6d0872f5d7b6cd4e60996ea5236603/site/app/views/grading/ElectronicGraderView.php#L1063-L1068

and so the proper way to write it would be:

```php
         return $return . <<<HTML 
                                              </div> 
                                          </div> 
                                      </div> 
                          		</div> 
HTML; 
```

**Review by shailpatels — APPROVED** [review]

**Comment by bmcutler** [comment]
@MasterOdin 
I committed your edit, but now its failing php linting???

**Pull request merged**